metrics.go

```diff
@@ -50,6 +51,22 @@ func flattenMetrics(data metrics, memo flatMetrics, prefix string) flatMetrics {
 	return memo
 }
 
+func decodeSyncData(data metrics, prefix string) flatMetrics {
+	var memo = make(flatMetrics)
+	for k, v := range data {
+		key := prefix + normalizeKey(k)
+		value := decodeHexAddr(v.(string))
+		memo[key] = value
+	}
+	return memo
+}
+
+func decodeHexAddr(s string) string {
+	/* Prometheus needs a number, not a hex */
+	number, _ := strconv.ParseUint(s, 0, 64)
```

Comment:
I would put something like `-1` if decoding fails so we can easily filter them out (or just filter by that to see where errors happen)

Reply:
That's a good point, thanks!